examples/sdk-examples/src/examples/basic/HeadlineExample.tsx

```diff
@@ -8,45 +8,43 @@ import * as Md from "../../md/full";
 const FranchiseFees = modifyMeasure(Md.$FranchiseFees, (m) => m.format("#,##0").title("Franchise Fees"));
 const FranchiseFeesAdRoyalty = modifyMeasure(Md.$FranchiseFeesAdRoyalty, (m) => m.format("#,##0"));
 
-export class HeadlineExample extends Component {
-    public onLoadingChanged: OnLoadingChanged = (...params) => {
+export const HeadlineExample: React.FC = () => {
+    const onLoadingChanged: OnLoadingChanged = (...params: any[]) => {
         // eslint-disable-next-line no-console
         return console.log("ColumnChartExample onLoadingChanged", ...params);
```

Comment:
Should we get rid of these console logs? I don't see any big added value to include them in the examples... I would keep examples as simple as possible. @no23reason What do you think? Any reason to keep them?